content/doc/book/updating/linux.adoc

```diff
@@ -0,0 +1,102 @@
+---
+layout: section
+title: Linux
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc: left
+:toclevels: 3
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+link:https://www.jenkins.io/doc/book/installing/linux/[Linux installations]
+should be upgraded using the same method as was used for the original installation
+This section will guide you through updating your Jenkins instance to the most recent version.
+
+include::doc/book/updating/_pre-upgrade-wizard.adoc[]
+
+== Upgrade steps on Linux
+[unordered.stack]
+1) Stop your Jenkins Instance :: 
+[source,bash]
+```
+#Stop running Jenkins service using the following command:
+
+sudo su -
+service jenkins stop
+```
+2) Take Jenkins instance backup as a recovery option ::
+[source,bash]
+```
+#Take backup of Jenkins home directory. In my case my Jenkins home directory is /var/lib/jenkins.
+
+cd /var/lib
+tar -cvzf jenkins_date.tar.gz jenkins/
+mv jenkins_date.tar.gz $HOME
+```
+
+3) Take The backup of the current Jenkins war file ::
+[source,bash]
+```
+# Take the backup of Jenkins current version binary using following commands:
+
+cd /usr/share/java/
+cp -p jenkins.war jenkins.war.old
+```
+
+4) Download The New LTS Jenkins Version ::
+[source,bash]
+```
+#Download the new version of Jenkins WAR file using the following commands:
+
+wget https://updates.jenkins-ci.org/latest/jenkins.war
+```
+
+5) Start The Jenkins Service ::
+[source,bash]
+```
+#Start the Jenkins service using the following command:
+service jenkins start
+```
+
+== Updating Based on Operating Systems
+---
+
+Red Hat / CentOS ::
+Beginning in Jenkins version 2.319, the epel-release package is required to upgrade Jenkins using an RPM.
+However, some CentOS distributions do not include it.
+For those CentOS distributions, run the following command before you begin the installation procedure: 
+`sudo yum install epel-release`.
+
+* Update Controller with `yum upgrade jenkins`
+
+* Update Operations center with `yum upgrade jenkins-oc`
+
+RPM/YUM with High Availability (Important) ::
+
+The RPM package contains a post-install script to ensure ownership on several files including JENKINS_HOME.
+Therefore if High Availability is setup, this could lead to I/O Errors - for example when upgrading one node while another one is still running.
+More information is available in link:https://issues.jenkins.io/browse/JENKINS-23273[JENKINS-23273].
+
+To workaround this problem, you can skip this script by adding the property
+[source]
+```
+JENKINS_INSTALL_SKIP_CHOWN="true"
+```
+under `/etc/sysconfig/jenkins`.
+
+Debian / Ubuntu ::
+* Update Controller with `apt-get update && apt-get install jenkins`
+
+* Update Operations center with `apt-get update && apt-get install jenkins-oc`
```

Comment:
Jenkins does not have anything called Operations Center.  That is a part of the CloudBees products.  We rely on CloudBees to document their products.
```suggestion
```